### PR TITLE
Update getting_started.md

### PR DESCRIPTION
**Pull request by devoncarew**
1. Make style consistent (**menu>submenu** vs `menu > submenu`)
1. Move Flutter SDK install to the to make sure it's there for plugin config
1. Add missing plugin SDK config step
1. Change initial step to open the hello world app so that there is something to run

### CODE REVIEW

File: getting_started.md [hunk 1]
```diff
@@ -31,17 +36,26 @@ can be installed, the repository needs to be added to the IDE.
...
 
 ### Running an app
 
 Ensure a development device is available and running (a development enabled Android device, or the
 iOS Simulator). In the IntelliJ toolbar, select the launch configuration for the created Flutter 
 project and hit the debug button.
+
+### Additional details
+
+For tips on opening existing projects, debugging, etc. see the [user journeys page](/docs/user_journeys.md).
```
Comment (devoncarew) [inline getting_started.md]: Are our user journeys meant to be user facing? As in getting started instructions, vs. UI paths to guide our development?

Comment (devoncarew) [inline getting_started.md]: Let's land as is, but plan to in-line getting started UI tasks into our user-facing docs, and have the user journeys serve a dual purpose of guiding development efforts, and acting as smoke test scripts for our releases.

**Comment by devoncarew** [comment]
lgtm!

File: getting_started.md (hunk 1 above)
Comment (stevemessick) [inline getting_started.md]: Yeah, I thought user journeys were internal docs.

**Comment by devoncarew** [comment]
(landing...)

**Pull request merged**